license.go

```diff
@@ -30,16 +32,23 @@ type LicenseService struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/license.html
 type License struct {
-	StartsAt  *ISOTime `json:"starts_at"`
-	ExpiresAt *ISOTime `json:"expires_at"`
-	Licensee  struct {
+	ID               int        `json:"id"`
+	Plan             string     `json:"plan"`
+	CreatedAt        *time.Time `json:"created_at"`
+	StartsAt         *ISOTime   `json:"starts_at"`
+	ExpiresAt        *ISOTime   `json:"expires_at"`
+	HistoricalMax    int        `json:"historical_max"`
+	MaximumUserCount int        `json:"maximum_user_count"`
+	Expired          bool       `json:"expired"`
+	Overage          int        `json:"overage"`
+	UserLimit        int        `json:"user_limit"`
+	ActiveUsers      int        `json:"active_users"`
+	Licensee         struct {
 		Name    string `json:"Name"`
 		Company string `json:"Company"`
 		Email   string `json:"Email"`
 	} `json:"licensee"`
-	UserLimit   int `json:"user_limit"`
-	ActiveUsers int `json:"active_users"`
-	AddOns      struct {
+	AddOns struct {
 		GitLabFileLocks int `json:"GitLabFileLocks"`
```

Comment:
Hmm... I know you didn't added this one, but it caught my eye... Would you mind fixing this one while your changing this struct?
```suggestion
		GitLabFileLocks   int `json:"GitLab_FileLocks"`
		GitLabAuditorUser int `json:"GitLab_Auditor_User"`
```